share/completions/az.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_az_complete
+  set -x _ARGCOMPLETE 1
+  set -x _ARGCOMPLETE_IFS \n
+  set -x _ARGCOMPLETE_SUPPRESS_SPACE 1
+  set -x _ARGCOMPLETE_SHELL fish
+  set -x COMP_LINE (commandline -p)
+  set -x COMP_POINT (string length (commandline -cp))
+  set -x COMP_TYPE
+  if set -q _ARC_DEBUG
+    az 8>&1 9>&2 1>/dev/null 2>&1
+  else
+    az 8>&1 9>&2 1>&9 2>&1
```

Comment:
What the hedge is that fd awkwardness? If I read this correctly it swaps stdout and stderr around. Do they really send the actual requested output to stderr and errors to stdout?

If so, just redirect stderr to stdout and silence stdout:

```fish
az 2>&1 >/dev/null
```

Or do they actually expect fd 8 and 9 to be set?

What happens if you leave it off?

Reply:
That's actually needed. I found no difference when I left of the third redirection and I've silenced stderr, but the other ones are really needed for az to provide completions correctly